<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SphinxQt 1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="layoutCentral">
    <item row="3" column="0" colspan="3">
     <layout class="QHBoxLayout" name="layoutControls">
      <item>
       <widget class="QPushButton" name="buttonInitialize">
        <property name="text">
         <string>Инициализация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDestroy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сброс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelIcon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/root/Resources/norecord.ico</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="editText"/>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QFormLayout" name="layoutForm">
      <item row="0" column="0">
       <widget class="QLabel" name="labelDevices">
        <property name="text">
         <string>Имя устройства:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboDevices"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelRates">
        <property name="text">
         <string>Частота дискретизации:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboRates"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelAudio">
        <property name="text">
         <string>Путь к звуковой модели:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="layoutAudio">
        <item>
         <widget class="QLineEdit" name="editAudio">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonAudio">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelLang">
        <property name="text">
         <string>Файл языковой модели:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="layoutLang">
        <item>
         <widget class="QLineEdit" name="editLang">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonLang">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDict">
        <property name="text">
         <string>Файл словаря:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="layoutDict">
        <item>
         <widget class="QLineEdit" name="editDict">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonDict">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QPushButton" name="buttonStart">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Начать распознавание</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе SphinxQt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
